sr_unix/iopi_open.c

```diff
@@ -323,6 +328,7 @@ short iopi_open(io_log_name *dev_name, mval *pp, int fd, mval *mspace, int4 time
 #endif
         char		dev_name_buf[LOGNAME_LEN];
 	mstr		dev_mstr;
+	gtm_string_t    filtered_command;
```

Comment:
We have almost banned gtm_string_t from the YDB code base so this one too should likely be changed to ydb_string_t.